dali/python/nvidia/dali/_utils/external_source_impl.py

```diff
@@ -65,6 +65,10 @@ def __str__(self) -> str:
     "Unsupported callback return type. Expected NumPy array, PyTorch or MXNet cpu tensors, "
     "DALI TensorCPU, list of those types or DALI TensorListCPU representing batch. Got `{}` instead.")
 
+_tf_uniform_error_msg = (
+    "Unsupported callback return value. TensorFlow requires that the batches produced by input "
+    "datasets or ExternalSource `source` callback in batch mode (that is when batch=True) "
```

Comment:
```suggestion
    "datasets or external source `source` callback in batch mode (that is when batch=True) "
```
I think we no longer use `ExternalSource` as an operator name in the docs.

Reply:
Also I'm not sure if we have TF specific message here.

Reply:
I'm still capitalizing ops names, unless I write specifically fn.some_operator. But probably that is also not consistent within the codebase.